src/compose/ComposeBox.js

```diff
@@ -357,6 +376,12 @@ class ComposeBox extends PureComponent<Props, State> {
 
     return (
       <View style={this.styles.wrapper}>
+        {showDeactivatedGroupPMWarning && (
+          <Label
+            text="One or more recipients may have been deactivated"
```

Comment:
```diff
- "One or more recipients may have been deactivated"
+ "One or more recipients have been deactivated"
```
I think? It's not uncertain: we know that at least one has been deactivated.

Reply:
Yep, fixed.